lib/calendarium-romanum/temporale.rb

```diff
@@ -354,30 +354,46 @@ def sunday(date)
     def ferial(date)
       seas = season date
       week = season_week(seas, date)
+      week_ord = Ordinalizer.ordinal week
       rank = Ranks::FERIAL
       title = nil
+      id = nil
       case seas
       when Seasons::ADVENT
         if date >= Date.new(@year, 12, 17)
           rank = Ranks::FERIAL_PRIVILEGED
           nth = Ordinalizer.ordinal(date.day)
-          title = I18n.t 'temporale.advent.before_christmas', day: nth
+          title = I18n.t 'temporale.advent.before_christmas', day: nth, week: week_ord, weekday: I18n.t("weekday.#{date.wday}")
+          id = "advent_#{(I18n.t "weekday.#{date.wday}", locale: :en).downcase}_december#{date.day}"
         end
       when Seasons::CHRISTMAS
         if date < mother_of_god
           rank = Ranks::FERIAL_PRIVILEGED
 
-          nth = Ordinalizer.ordinal(date.day - nativity.day + 1) # 1-based counting
+          day_of_octave = date.day - nativity.day + 1 # 1-based counting
+          nth = Ordinalizer.ordinal(day_of_octave)
           title = I18n.t 'temporale.christmas.nativity_octave.ferial', day: nth
+          id = "christmas_octave_#{day_of_octave}"
         elsif date > epiphany
-          title = I18n.t 'temporale.christmas.after_epiphany.ferial', weekday: I18n.t("weekday.#{date.wday}")
+          if transferred_to_sunday?(:epiphany)
```

Comment:
@bbloomf this code would only apply to the US correct?

Reply:
I'm not sure exactly which code you're talking about, but I tried to keep it universal, and nothing should be applicable only to the US, but rather wherever Epiphany is always celebrated on Sunday